### PR TITLE
add auto-tokens argument back

### PR DESCRIPTION
**Pull request by granawkins**
Short PR to add the `auto-tokens` arg back, with a default of 4000, which applies only to additional files added by `auto-context`, as discussed.

This is quite necessary now that `gpt-4-1103` is the default model, so I'm doing a tiny PR to fast-track it.

### CODE REVIEW

File: mentat/config.py [hunk 1]
```diff
@@ -83,6 +83,13 @@ class Config:
         },
         converter=converters.optional(converters.to_bool),
     )
+    auto_tokens: int = attr.field(
+        default=4000,
```
Comment (jakethekoenig) [inline mentat/config.py:87]: I think the default should be a little higher. Like 8000. The one benchmark I wrote fails when autocontext is this low but succeeds at 8000. Sorry I wish we had more benchmarks already so we could be more scientific.

File: mentat/config.py [hunk 2]
```diff
@@ -83,6 +83,13 @@ class Config:
...
+        metadata={
+            "description": "The number of tokens auto-context can add.",
```
Comment (jakethekoenig) [inline mentat/config.py:89]: Maybe rephrase this to "will add" since that's the current behavior roughly

**Review by jakethekoenig — APPROVED** [review]
Thanks!

File: mentat/config.py (hunk 2 above)
Comment (granawkins) [reply mentat/config.py:89]: Only if it's turned on though, that was the 'can'

File: mentat/config.py (hunk 1 above)
Comment (granawkins) [reply mentat/config.py]: ya I don't know either. I'll set it to 8k and lets keep an eye on it.

File: mentat/config.py (hunk 2 above)
Comment (granawkins) [reply mentat/config.py:89]: On second thought I agree, will is clearer.
